appyters/PrismEXP/static/AutocompleteField.js

```diff
@@ -0,0 +1,118 @@
+// Adapted from W3schools
+
+function runall($, name, file_path) {
```

Comment:
If you're not using this field, you can remove this file artifact.

Reply:
also not used